gpflow/kernels/mo_kernels.py

```diff
@@ -164,6 +180,11 @@ def __init__(self, kernels, W, name=None):
     def num_latent_gps(self):
         return self.W.shape[-1]  # L
 
+    @property
+    def kernels(self):
+        """The underlying kernels in the multioutput kernel"""
+        return self._kernels
+
```

Comment:
Same- can this be deleted now ?
```suggestion
```

Reply:
no I dont think so. I'm happy to add the property to the Combination if you'd prefer but we should keep it as a property

Reply:
take it back its in the base! oops!